.github/workflows/cli-release.yml

```diff
@@ -12,78 +12,48 @@ name: Release CLI
 
 jobs:
   build:
-    name: Build ${{ matrix.os }}-${{ matrix.architecture }}
+    name: Build ${{ matrix.architecture }}-${{ matrix.target-suffix }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        include:
-          # Linux x86_64 build on Ubuntu
-          - os: ubuntu-latest
-            architecture: x86_64
-            suffix: unknown-linux-gnu
-
-          # TODO: Linux aarch64 (ARM64) build on Ubuntu via cross-compilation - probably use `cross-rs`
-
-          # macOS x86_64 build
-          - os: macos-latest
-            architecture: x86_64
-            suffix: apple-darwin
-
-          # macOS ARM64 build
-          - os: macos-latest
-            architecture: aarch64
-            suffix: apple-darwin
+          os: [ ubuntu-latest, macos-latest ]
+          architecture: [ aarch64, x86_64 ]
+          include:
+            - os: ubuntu-latest
+              target-suffix: unknown-linux-gnu
+            - os: macos-latest
+              target-suffix: apple-darwin
 
     steps:
-      # Step 1: Checkout the code
       - name: Checkout code
         uses: actions/checkout@v4
 
-      # Step 2: Set up Rust toolchain
-      # This will install the specified target as per matrix
       - name: Set up Rust toolchain
-        uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
The toolchain  `actions-rust-lang/setup-rust-toolchain@v1` does not with cross. Not sure why.  

We can use hermit in the future to set up the rust so that we have consistent environment locally and the build environment

Reply:
yeah that is fine